externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/PlanOnlyStatement.scala

```diff
@@ -28,6 +28,7 @@ import org.apache.spark.sql.types.StructType
 import org.apache.kyuubi.KyuubiSQLException
 import org.apache.kyuubi.config.KyuubiConf.{LINEAGE_PARSER_PLUGIN_PROVIDER, OPERATION_PLAN_ONLY_EXCLUDES, OPERATION_PLAN_ONLY_OUT_STYLE}
 import org.apache.kyuubi.engine.spark.KyuubiSparkUtil.getSessionConf
+import org.apache.kyuubi.engine.spark.operation.planonly.SQLStringifyPlugins
```

Comment:
So the plugin is actually a must for spark sql engine？ why do we need to separate it from the engine module？

Reply:
> So the plugin is actually a must for spark sql engine？ why do we need to separate it from the engine module？

Because I don't want to introduce kyuubi engine dependency in kyuubi spark extensions

Reply:
we have to bundle this when packaging

Reply:
Yes, the kyuubi server plugin also works this way

Reply:
This plugin will be lightweight enough to only retain some interface definitions

Reply:
make sense